tests/unit/StoreActions.spec.ts

```diff
@@ -6,7 +6,7 @@ describe('Business Actions', () => {
 
   it('loads dissolution state filing', async () => {
     // init store properties we need
-    store.commit('stateFiling', 'dummy_url')
+    store.commit('setStateFiling', 'dummy_url')
```

Comment:
This and the line below were also bugs: they set the wrong store property. Because of this, the `loadStateFiling()` action resolved to null, but since there was no return statement, the call to fetch a filing was still made.

Anyway, it's all ironed out now.